dsms/knowledge/ktype.py

```diff
@@ -15,28 +32,121 @@ class KType(BaseModel):
     name: Optional[str] = Field(
         None, description="Human readable name of the KType."
     )
-    webform: Optional[Any] = Field(None, description="Form data of the KItem.")
+    webform: Optional[Webform] = Field(
+        None, description="Form data of the KType."
+    )
     json_schema: Optional[Any] = Field(
-        None, description="OpenAPI schema of the KItem."
+        None, description="OpenAPI schema of the KType."
+    )
+    rdf_mapping: Optional[str] = Field(None, description="")
```

Comment:
Also a description of the field is needed here. Maybe something like `"Optional rdf mapping for webform"`.